pypeclub/OpenPype
# Flame: OpenTimelineIO Export Modul

## Brief description
OpenTimelineIO implementation for exporting from Flame.

## Description
Implementing basic functionality for exporting otio_timeline, otio_clip, otio_track, otio_stack, otio_reference, otio_effect, etc.

## Testing notes:
1. create a project in ftrack/openpype db 
2. create folder in root of the project `editorial`, save
3. create task in the editorial folder `conforming`, save
4. sync to avalon
5. activate Flame on project in application custom attributes
6. On the machine with Flame 2021 installed click on flame launcher
7. Flame should open into project. If project is not available in wiretap it will be created automatically, if user is not created, it will be created automatically - taken from os user name.
8. create a sequence in reels and open it in timeline editor
9. select couple clips and right button click > OpenPype > Publish
10. publish window will open and in log of `test_collector` plugin should be printed info about OpenTimelineIO timeline created
